### PR TITLE
Update documentation on Pricing

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

**Review by nilminiwso2 — APPROVED** [review]

File: en/docs/reference/pricing.md [hunk 1]
```diff
@@ -2,229 +2,142 @@
...
 
-An organization has a flight booking system on Choreo. This system involves several APIs and databases. The scenario and the APIs being used in the system are described below.
+    The infrastructure cost consists of the charges to Choreo by the cloud platform for vCPU, RAM, storage, ingress, egress, and network. Choreo provides $100 worth of infrastructure free of charge to each organization.
 
-#### Scenario
+    If the infrastructure cost of an organization on the Free plan exceeds $100, Choreo will limit some of the services it provides to it (e.g., some components might be deactivated).
 
-To make a booking through the web application, a client first searches for available flights and then proceeds to make the booking. The client application (web application) upon receiving the request makes a call to the [Search API](#searchAPI) to search for available flights. When the client receives the search results the client proceeds to make a reservation using the [Reservations API](#reservationsAPI).
+    If the infrastructure cost of an organization on the Pay As You Go plan exceeds $100, it can pay the additional infrastructure charges to the Choreo Sales Team.
```
Comment (AyeshW) [inline en/docs/reference/pricing.md:19]: What do you mean by paying to `Choreo Sales Team` for Pay As You Go plans? We do payments the cloud vendor.

File: en/docs/reference/pricing.md [hunk 2]
```diff
@@ -2,229 +2,142 @@
...
 
-The following two diagrams illustrate the interactions in the system while searching and reserving a flight.
-<br/>
+    If the infrastructure cost of an organization on the Enterprise plan exceeds $100, it can pay the additional infrastructure charges to the Choreo Sales Team or directly to the cloud platform vendor.
```
Comment (AyeshW) [inline en/docs/reference/pricing.md]: For the enterprise plans, payments are handled by the sales team, not the cloud vendor (i.e. Stripe, Azure). @isuruw please verify.

File: en/docs/reference/pricing.md (hunk 1 above)
Comment (Shehanir) [inline en/docs/reference/pricing.md:19]: Infrastructure cost of Pay As You Go subscriptions will be charged directly from the billing portal. will not go through the sales team.

File: en/docs/reference/pricing.md (hunk 2 above)
Comment (Shehanir) [inline en/docs/reference/pricing.md:21]: infrastructure cost and the cost of the component usage of an organization on the Enterprise plan will only handle by the sales team not going through the billing portal.

File: en/docs/reference/pricing.md [hunk 3]
```diff
@@ -2,229 +2,142 @@
...
 
-### Example 2: REST API Proxy
-An organization exposes a flight tracking API as a managed API on Choreo. This system involves several APIs and databases. The scenario and the APIs in the system are described below:
+Choreo allocates up to a maximum of 2x vCPUs and 4 GB RAM infrastructure resources to your component workloads.
 
-#### Scenario
+**Pay As You Go**
 
-To search for real-time flight data, a client enters a search criteria, and searches through the web application. Upon receiving the request, the client application (web application) makes a call to the [Flights Status API](#flightStatusAPI), a managed API on Choreo, to search for flight data. Choreo then invokes the backend API, the [Real-Time Flight API](#realtimeFlightAPI). The backend API then invokes a database to get the information and passes it to the client.
+This plan requires you to pay $150 per component (except for the first five components) and pay for any additional infrastructure costs incurred (i.e., if the infrastructure cost is over $100). It does not require you to pay an annual fee. You can deploy up to 25 components (i.e., any combination of component types).
```
Comment (AyeshW) [inline en/docs/reference/pricing.md:36]: Shouldn't we mention that this $150 per component is monthly basis?

File: en/docs/reference/pricing.md (hunk 1 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Changed as specified.

Comment (RukshiW) [reply en/docs/reference/pricing.md]: Changed as specified.

File: en/docs/reference/pricing.md (hunk 2 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Changed as specified.

**Review by AyeshW — CHANGES_REQUESTED** [review]

File: en/docs/reference/pricing.md [hunk 4]
```diff
@@ -2,229 +2,142 @@
...
+This plan allows you to pay an annual subscription of $50,000 or more (as agreed) and deploy up to 50 components (i.e., any combination of component types). If you need to deploy over 50 components, you can renegotiate with Choreo to pay an agreed amount per additional component or revise the annual subscription.
 
-- <a class="pricing" id="flightsDatabase">Flights Database</a>: A database that stores flight information.
+The following table summarizes the cost of each plan:
 
+ |                      | **Free**       | **Pay As You Go** | **Enterprise** |
+ |----------------------|----------------|-------------------|----------------|
+ | Subscription    | Free subscription  | Based on the number of components| Minimum $50k² for the annual commitment. The annual commitment includes 50x components (any combination of component types). |
+ | Infrastructure  | Up to $100 worth of infrastructure services free of charge. | Up to $100 worth of infrastructure services free of charge.<br/><br/> | Pass through or pay directly to Azure. |
+ | Monthly Credits | $100               | N/A | N/A|
```
Comment (Shehanir) [inline en/docs/reference/pricing.md:50]: This monthly credits of $100 will be given for the Infrastructure usage. So I think we should remove this line.

File: en/docs/reference/pricing.md (hunk 2 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Changed this.

File: en/docs/reference/pricing.md (hunk 4 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Done

File: en/docs/reference/pricing.md (hunk 3 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Changed as follows:

```
This plan requires you to pay the following:
- $150 per component (except for the first five components) on a monthly basis
- Any additional infrastructure costs incurred (i.e., if the infrastructure cost is over $100). 

This plan does not require you to pay an annual fee. You can deploy up to 25 components (i.e., any combination of component types). 
```

**Review by isuruw — DISMISSED** [review]
LGTM

File: en/docs/reference/pricing.md [hunk 5]
```diff
@@ -2,229 +2,145 @@
...
-
-### What is a step?
-A step (1 step) is a maximum of 500ms compute time on a container having 350MB of memory and 0.5 vCPU. Any additional compute time is counted as an additional step.  
-An incoming event or message to Choreo or an outgoing API call from Choreo is a minimum of 1 step.
+The pricing for your Choreo deployment is mainly determined based on the following:
 
+- **The number of components**
 
-## Pricing examples
+    You can have up to five free components ((i.e., any combination of component types), and pay for each additional component.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/reference/pricing.md:11]: what do you mean by combination of component types?

File: en/docs/reference/pricing.md [hunk 6]
```diff
@@ -2,229 +2,145 @@
...
 
-- **Search process**: Single execution of a search would cost a total of **2 steps**. 
-- **Reservation process**: Single execution of reservation process would cost a total of **6 steps**. 
-- **Total**: A single execution of the search and reservation process of a flight would cost an organization a total of **8 steps in total**.
+You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you want to deploy over 5 components.
 
-!!! note
-    The above cost calculation is dependent on the above architecture and design of the APIs.
-<br/>
+Choreo allocates up to a maximum of 2x vCPUs and 4 GB RAM infrastructure resources to your component workloads.
```
Comment (shaniR) [inline en/docs/reference/pricing.md:32]: Shouldn't we mention that we give 5 components in the free tier?

File: en/docs/reference/pricing.md [hunk 7]
```diff
@@ -2,229 +2,145 @@
...
+This plan does not require you to pay an annual fee. You can deploy up to 25 components (i.e., any combination of component types). 
 
-#### Glossary
+If the number of components you need to deploy exceeds 25 or if your annual spend exceeds $50,000, we recommend that you change your pricing plan to Enterprise.
 
-- <a class="pricing" id="flighStatusAPI">Flight Status API</a>: An external-facing, managed API on Choreo. This API exposes the Real-Time Flight API on Choreo as a managed API.
+**Enterprise** 
 
-- <a class="pricing" id="realtimeFlightAPI">Real-TIme Flight API</a>: An unmanaged, external-facing API hosted outside of Choreo.
+This plan allows you to pay an annual subscription of $50,000 or more (as agreed) and deploy up to 50 components (i.e., any combination of component types). If you need to deploy over 50 components, you can renegotiate with Choreo to pay an agreed amount per additional component or revise the annual subscription.
```
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/reference/pricing.md:46]: Shouldn't we say contact Choreo sales or something like that, rather then telling them they can renegotiate?

File: en/docs/reference/pricing.md [hunk 8]
```diff
@@ -2,229 +2,145 @@
...
-- <a class="pricing" id="flightsDatabase">Flights Database</a>: A database that stores flight information.
+The following table summarizes the cost of each plan:
 
+ |                      | **Free**       | **Pay As You Go** | **Enterprise** |
+ |----------------------|----------------|-------------------|----------------|
+ | Subscription    | Free subscription  | Based on the number of components| Minimum $50k² for the annual commitment. The annual commitment includes 50x components (any combination of component types). |
+ | Infrastructure  | Up to $100 worth of infrastructure services free of charge. | Up to $100 worth of infrastructure services free of charge.<br/><br/> | Pass through or pay directly to Azure. |
 
-#### Get flight status
+The following table summarizes the usage of each plan:
```
Comment (shaniR) [inline en/docs/reference/pricing.md:55]: duplicate sentence

**Review by AyeshW — DISMISSED** [review]

**Review by Shehanir — APPROVED** [review]

File: en/docs/reference/pricing.md [hunk 9]
```diff
@@ -2,229 +2,145 @@
...
 
-![Get flight status](../assets/img/references/api-proxy-example1.png)
+ |                      | **Free**       | **Pay As You Go** | **Enterprise** |
+ |----------------------|----------------|-------------------|----------------|
+ | **No of Projects**   | 1              | Unlimited         | Unlimited      |
+ | **No of Components** | A maximum of 5 | A maximum of 25 (recommended)  | Unlimited      |
+ | **No of Developers** | Unlimited      | Unlimited         | Unlimited      |
 
-Let’s see how the steps are calculated for the process to obtain the flight status:
+### Features
```
Comment (shaniR) [inline en/docs/reference/pricing.md:63]: Since all features are supported by all plans, shall we have that sentence as a common sentence?

File: en/docs/reference/pricing.md [hunk 10]
```diff
@@ -2,229 +2,145 @@
...
 
-- <a class="pricing" id="aircraftNotifWebhook">Aircraft Status Notifier</a>:A webhook in Choreo that listens to any events that trigger when the Aircraft maintenance sheet is updated.
-- <a class="pricing" id="AircraftMaintenanceSheet">Aircraft Maintenance Sheet</a>: A Google sheet that maintains the list of aircraft, their information, the maintenance schedule, and the maintenance status.
-- <a class="pricing" id="flightSchedulingDatabase">Flight Scheduling Database</a>:The database that includes details about  flights, the aircraft assigned to the flight route, and their maintenance state.
+- Deployments to cloud platforms are supported for the three plans as follows:
 
-<br/>
+   | **Free** | **Pay As You Go** | **Enterprise** |
+   |----------|-------------------|----------------|
+   | Azure    | Azure<br/>AWS<br/>GCP<br/>VMWare<br/>OpenShift<br/>Tanzu<br/>Any Kubernetes cluster |Azure<br/>AWS<br/>GCP<br/>VMWare<br/>OpenShift<br/>Tanzu<br/>Any Kubernetes cluster |
```
Comment (shaniR) [inline en/docs/reference/pricing.md:81]: The table in the preview mode looks weird. Shall we have like bullet points inside the table?

File: en/docs/reference/pricing.md [hunk 11]
```diff
@@ -2,229 +2,145 @@
...
+#### Integration and automation
 
-Let’s see how the steps are calculated for the update maintenance process:
+Choreo supports the following integration and automation features for all the plans:
 
-- As illustrated above, the webhook registers with the Google sheet at the time the Webhook component is created on Choreo.
-- The supervisor initiates the update process by updating or adding the relevant information on the [Aircraft Maintenance Sheet](#AircraftMaintenanceSheet).
-- The Google Sheet triggers **1 event to the Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook)** on Choreo. This is counted as **1 step**.
-- The Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook) on Choreo then makes **1 database call to the [Flight Scheduling Database](#flightSchedulingDatabase)** to update the flight maintenance status. This is counted as 1 step. 
+- Triggers
```
Comment (shaniR) [inline en/docs/reference/pricing.md:88]: Are these components or features?

File: en/docs/reference/pricing.md [hunk 12]
```diff
@@ -2,229 +2,145 @@
...
-An airline revises its ticket prices based on the price of jet fuel. This typically happens once a day. The airline uses a scheduled task on Choreo to run this job and to update the jet fuel price once a day.
+#### API management
 
-#### Scenario 
- 
-Airlines compute ticket prices typically once per day. The ticket prices depend on a number of factors. The price of oil is one such major factor that affects the pricing.  
+Choreo supports the following API management features for all the plans:
 
-The [scheduler](#scheduler) on Choreo runs once per day and fetches the oil prices for the day from the [Oil Price API](#oilPriceAPI). It then computes the jet fuel price applying the price calculation formula. The scheduler then updates the daily jet fuel price in the [Air Fare Database](#airFareDatabase).
+| Free           |  Pay As You Go  | Enterprise |
```
Comment (shaniR) [inline en/docs/reference/pricing.md:105]: The alignemnt of the rendered table is a little off

File: en/docs/reference/pricing.md [hunk 13]
```diff
@@ -2,229 +2,145 @@
...
+|----------------|-----------------|------------|
+| Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights<br/>(with only 7 days of data retention) | Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights | Creating new APIs<br/><br/>Securing APIs<br/><br/>Testing APIs<br/><br/>API Developer Portal<br/><br/>API Analytics and Insights |
```
Comment (shaniR) [inline en/docs/reference/pricing.md:107]: How many days data retention for other plans?

File: en/docs/reference/pricing.md [hunk 14]
```diff
@@ -2,229 +2,145 @@
...
-- <a class="pricing" id="oilPriceAPI">Oil Price API</a>: An external-facing API that returns the crude oil price for the day.
-- <a class="pricing" id="airFareDatabase">Air Fare Database</a>: The database that stores the jet fuel price which is used to compute the airfare for each route.
-- <a class="pricing" id="airlineRoutesAPI">Airline Routes API</a>: An external-facing API that provides details about the airline routes and a list of airline routes.
-- <a class="pricing" id="airlineRoutesDatabase">Airline Routes Database</a>: The database that stores the airline routes information.
+Choreo supports monitoring and observability features for each plan as follows:
 
-<br/>
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| In-built monitoring dashboard<br/><br/>Logs<br/>(7 days of data retention)<br/><br/>Traces<br/>(7 days of data retention)<br/><br/>Alerting<br/>(7 days of data retention) | In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/>Alerting<br/><br/>Extending to 3rd Parties | In-built monitoring dashboard<br/><br/>Logs<br/><br/>Traces<br/><br/>Alerting<br/><br/>Extending to 3rd Parties |
```
Comment (shaniR) [inline en/docs/reference/pricing.md:116]: The rendered table is a little off

Comment (shaniR) [reply en/docs/reference/pricing.md:116]: Other plans how many days of data retention?

File: en/docs/reference/pricing.md [hunk 15]
```diff
@@ -2,229 +2,145 @@
...
 
-#### Update jet fuel price daily
+#### Security
 
-![ Update jet fuel prices daily](../assets/img/references/scheduled-task-example1.png)
+Choreo supports the following security features for all the plans:
 
-Let’s see how we can calculate the steps for the update maintenance process:
+- SSO for API Developer Portal
+- Enterprise Login
```
Comment (shaniR) [inline en/docs/reference/pricing.md:123]: should login be in lower case?

File: en/docs/reference/pricing.md [hunk 16]
```diff
@@ -2,229 +2,145 @@
...
+Choreo supports cloud-code editing for the three plans as explained below:
 
-- **Assuming the entire execution time for the update process takes 1200 ms**, the total number of steps can be calculated as follows:
-  
-        Number of API calls/database calls/events = 2 steps
-        Number of computation units = round((1200-500)/500) = 2 units = 2 steps
-        Total number of steps = 2 steps + 2 steps = 4 steps
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| 10 hours on the Choreo Web Editor. | 10 hours are allowed on the Choreo Web Editor per component free of charge.<br/><br/>If you need more code editing time, contact our Sales team to request a fully managed Cloud Coding Cluster. | If you have chosen the Enterprise plan, you may have a Cloud Coding Cluster already deployed on your infrastructure.<br/><br/>If you do not already have a Cloud Coding Cluster and you need one, contact our Sales team. |
```
Comment (shaniR) [inline en/docs/reference/pricing.md:134]: 10 hours are allowed on the Choreo Web Editor per component free of charge. -> 10 hours on the Choreo Web Editor per component free of charge.
Sales -> sales
Cloud Coding Cluster ->. do we need it in title case?

Comment (shaniR) [reply en/docs/reference/pricing.md:134]: Cloud Coding -> is it Google Cloud?

File: en/docs/reference/pricing.md [hunk 17]
```diff
@@ -2,229 +2,145 @@
...
-<br/>
+### Support
 
-#### Update airfare daily  
+Community support is available for all three plans. In addition, paid support is available for Pay As You Go and Enterprise plans.
 
-![ Update airfare daily](../assets/img/references/scheduled-task-example2.png)
+| Free           |  Pay As You Go  | Enterprise |
+|----------------|-----------------|------------|
+| Community support | Community support and paid plans | Community support and paid plans|
```
Comment (shaniR) [inline en/docs/reference/pricing.md:142]: How can someone get info for the paid plans for support? Shall we link them?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/reference/pricing.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md:11]: The five free components can be all REST APIs, 2 REST APIs and 3 webhooks, 4 webhooks and 1 scheduled task and so on.

File: en/docs/reference/pricing.md (hunk 14 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:116]: > The rendered table is a little off

It looks as follows in the local build:

Screen Shot 2023-01-05 at 12 00 12

File: en/docs/reference/pricing.md (hunk 12 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:105]: Though it appears that way in the Git preview, it looks as follows in the local build and will be rendered this way.

Screen Shot 2023-01-05 at 12 02 45

File: en/docs/reference/pricing.md (hunk 10 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:81]: Though it appears that way in the Git preview, it looks as follows in the local build and will be rendered this way.

Screen Shot 2023-01-05 at 12 07 32

File: en/docs/reference/pricing.md (hunk 9 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:63]: Although all features are supported, some features are supported with limitations based on the payment. I think having such a statement at the beginning can mislead.

File: en/docs/reference/pricing.md (hunk 8 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:55]: Can see only one occurrence. In which line is it duplicated?

File: en/docs/reference/pricing.md (hunk 11 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:88]: In this context, they are features.

File: en/docs/reference/pricing.md [hunk 18]
```diff
@@ -2,229 +2,145 @@
 
 ## Introduction
 
-Choreo is a Digital Platform as a Service that helps you develop integration solutions faster by reducing the coding effort with a low-code development platform. You can use Choreo by paying only for your usage: nothing more, nothing less. Choreo has payment plans for all types of users. You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you use more than 5,000 [steps](#what-is-a-step) per month. If you plan to use Choreo extensively, you can also opt for an Enterprise plan and enjoy special discount rates.
+Choreo is a Digital Platform as a Service that helps you develop integration solutions faster by reducing the coding effort with a low-code development platform. You can choose the appropriate payment plan based on your usage.
```
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Removed this sentence, but left the line break before the next sentence because it opens a new topic.

File: en/docs/reference/pricing.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md]: "Have" means in the created state. Changed accordingly.

File: en/docs/reference/pricing.md [hunk 19]
```diff
@@ -2,229 +2,145 @@
...
-           4 steps + 2 steps = 6 steps
+Choreo currently supports the following payment plans:
 
-We can calculate the **total number of steps for searching and reserving a flight** as follows:
+**Free**
 
-- **Search process**: Single execution of a search would cost a total of **2 steps**. 
-- **Reservation process**: Single execution of reservation process would cost a total of **6 steps**. 
-- **Total**: A single execution of the search and reservation process of a flight would cost an organization a total of **8 steps in total**.
+You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you want to deploy over 5 components.
```
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Updated this section. Please check.

File: en/docs/reference/pricing.md (hunk 6 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:32]: Updated this section and mentioned that. Please check.

File: en/docs/reference/pricing.md (hunk 15 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Corrected.

File: en/docs/reference/pricing.md [hunk 20]
```diff
@@ -2,229 +2,145 @@
...
-<br/>
+Choreo allocates up to a maximum of 2x vCPUs and 4 GB RAM infrastructure resources to your component workloads.
 
-### Example 2: REST API Proxy
-An organization exposes a flight tracking API as a managed API on Choreo. This system involves several APIs and databases. The scenario and the APIs in the system are described below:
+**Pay As You Go**
 
-#### Scenario
+This plan requires you to pay the following:
+- $150 per component (except for the first five components) on a monthly basis
```
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Corrected.

File: en/docs/reference/pricing.md (hunk 2 above)
Comment (isuruw) [reply en/docs/reference/pricing.md]: +1

File: en/docs/reference/pricing.md (hunk 14 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: > Other plans how many days of data retention?

Updated this based on the latest information received from Rukshan and TharinduW

File: en/docs/reference/pricing.md (hunk 17 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:146]: Linked the Billing Portal.

File: en/docs/reference/pricing.md (hunk 14 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: > Other plans how many days of data retention?

Updated

File: en/docs/reference/pricing.md [hunk 21]
```diff
@@ -2,229 +2,145 @@
...
+- **The number of components**
 
-## Pricing examples
+    You can have up to five free components ((i.e., any combination of component types), and pay for each additional component.
 
-### Example 1: REST API 
+- **Infrastructure cost**
 
-An organization has a flight booking system on Choreo. This system involves several APIs and databases. The scenario and the APIs being used in the system are described below.
+    The infrastructure cost consists of the charges to Choreo by the cloud platform for vCPU, RAM, storage, ingress, egress, and network. Choreo provides $100 worth of infrastructure free of charge to each organization.
```
[the remark this answers is not in the document]
Comment (isuruw) [reply en/docs/reference/pricing.md:15]: The infrastructure costing is not yet charged because the feature is not fully implemented. The infrastructure cost break down will be shown for PAYG and Enterprise customers on the billing portal and for free users we are yet to decide on how to show it. Therefore we don't need to include that information here as of now, and update the docs once the feature is completed, until then users are not charged with the Infra cost.

[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md:15]: Included the sentence "Currently, users on Pay As You Go and Enterprise plans can get their infrastructure costs from the cloud platform vendor."

Apparently, the Billing Portal will eventually provide a breakdown of the infrastructure costs, but this is not implemented yet.

File: en/docs/reference/pricing.md (hunk 13 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Updated.

File: en/docs/reference/pricing.md (hunk 16 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Removed title case. It is not Google Cloud.

Comment (RukshiW) [reply en/docs/reference/pricing.md]: > 10 hours are allowed on the Choreo Web Editor per component free of charge. -> 10 hours on the Choreo Web Editor per component free of charge. Sales -> sales Cloud Coding Cluster ->. do we need it in title case?

Done

File: en/docs/reference/pricing.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/reference/pricing.md]: Shall we change this to up to five free components of any type

File: en/docs/reference/pricing.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/reference/pricing.md:15]: ack. @RukshiW  shall we remove this section in that case?

File: en/docs/reference/pricing.md [hunk 22]
```diff
@@ -2,229 +2,150 @@
...
+## Plans
 
-- **Assuming the entire execution time for the reservation process takes 1200ms (1.2s)** , in addition to the above steps,  the total number of extra computation units consumed can be calculated as follows using ***floor((t)/500)) where t is the execution time***: 
+Choreo currently supports the following payment plans:
 
-          floor(1200/500) = 2 units = 2 steps
+**Free**
 
-Therefore the total number of steps for the reservation process is: 
+The free tier allows you to create up to five free components. These five components can be any combination of component types.
```
Comment (shaniR) [inline en/docs/reference/pricing.md:29]: Shall we rephrase this to: Allows you to create any five components.

File: en/docs/reference/pricing.md [hunk 23]
```diff
@@ -2,229 +2,150 @@
...
 
-           4 steps + 2 steps = 6 steps
+Choreo allocates up to a maximum of 2x vCPUs and 4 GB RAM infrastructure resources to your component workloads.
 
-We can calculate the **total number of steps for searching and reserving a flight** as follows:
+You can enjoy the free tier with limitations and then move on to the Pay-As-You-Go model if you want to create over five components.
```
Comment (shaniR) [inline en/docs/reference/pricing.md:33]: Pay-As-You-Go plan?

File: en/docs/reference/pricing.md (hunk 8 above)
Comment (shaniR) [reply en/docs/reference/pricing.md:56]: oh yes my bad!

File: en/docs/reference/pricing.md (hunk 11 above)
Comment (shaniR) [reply en/docs/reference/pricing.md:89]: It can be confusing, as we are referring to them as components, and when we come here it is features. Can give a wrong idea that Triggers and scheduled jobs are free

File: en/docs/reference/pricing.md (hunk 12 above)
Comment (shaniR) [reply en/docs/reference/pricing.md:106]: ack.

**Review by shaniR — CHANGES_REQUESTED** [review]

File: en/docs/reference/pricing.md (hunk 22 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:29]: Doesn't that sound vague?

File: en/docs/reference/pricing.md (hunk 23 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Changed

File: en/docs/reference/pricing.md (hunk 22 above)
Comment (shaniR) [reply en/docs/reference/pricing.md:29]: I am sorry, can you elaborate?

**Comment by RukshiW** [comment]
About comment "It can be confusing, as we are referring to them as components, and when we come here it is features. Can give a wrong idea that Triggers and scheduled jobs are free" on line 88.

Well, they are free as long as you don't exceed the component quota isn't it? @isuruw any comment on this?

File: en/docs/reference/pricing.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Done

[the remark this answers is not in the document]
Comment (isuruw) [reply en/docs/reference/pricing.md]: Lets not remove, the document should be in tally with the web site, this information is already there on the site https://wso2.com/choreo/pricing/. Therefore lets have similar information here and we will update with more information once the feature is completed. The feature is planned to be release by late January or by the first week of February 2023

File: en/docs/reference/pricing.md (hunk 16 above)
Comment (isuruw) [reply en/docs/reference/pricing.md]: @RukshiW @shaniR we need to remove this section and any where we have mentioned about Cloud Coding Cluster, this refer to our code editor which will get removed soon. Therefore we don't need this anymore. Lets keep a backup anyway and remove this section.

File: en/docs/reference/pricing.md [hunk 24]
```diff
@@ -2,229 +2,137 @@
...
 
-![ Update jet fuel prices daily](../assets/img/references/scheduled-task-example1.png)
+- SSO for API Developer Portal
+- Enterprise login
+- Container manifest scanning
+- Container static scanning
+- Run-time container security
 
-Let’s see how we can calculate the steps for the update maintenance process:
+#### Cloud-code editing
```
Comment (isuruw) [inline en/docs/reference/pricing.md:132]: @RukshiW we can remove this section. Lets keep a backup and remove.

File: en/docs/reference/pricing.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md:15]: Added it back.

[the remark this answers is not in the document]
Comment (shaniR) [reply en/docs/reference/pricing.md:15]: There are some points in the website that we have not added such as 

-Choreo infrastructure credits gives you approximately 1,000 vCPU hours to process your containerized workloads. Infrastructure costs include vCPU, RAM, storage, ingress and egress, and network charges.

- Infrastructure costs for each component (e.g., vCPU, RAM, network utilization - ingress/egress, and storage) come with zero markups. You can also use your own cloud provider and pay those costs directly.

And also add a note saying this is not yet charged and update when we do start charging. Else we will be misleading the users with a further documentation to pricing.

File: en/docs/reference/pricing.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Again, it is not specific enough and can leave room for someone to think that all five need to be of the same type, and that type can be any supported.

File: en/docs/reference/pricing.md (hunk 16 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Done

**Review by shaniR — APPROVED** [review]
Approving the PR as we want to keep it consistent with the UI. However, added a comment to make it more clear to the user It would be great to accommodate it at a later stage.

File: en/docs/reference/pricing.md (hunk 22 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md:30]: We might as well say it allows you to create 5 components and stop there. Just adding the word "any" does not value

The questions that we are trying to answer here can be "Should the components be of a specific type?" "Should all components have the same type?", or "Is this quota per type". The sentence "Allows you to create any five components." does not give any context for the user to relate such a statement to such questions.

File: en/docs/reference/pricing.md (hunk 24 above)
Comment (RukshiW) [reply en/docs/reference/pricing.md]: Done

**Comment by RukshiW** [comment]
> > About comment "It can be confusing, as we are referring to them as components, and when we come here it is features. Can give a wrong idea that Triggers and scheduled jobs are free" on line 88.
> > Well, they are free as long as you don't exceed the component quota isn't it? The idea is that the trigger and scheduled job functionality is available for all payment plans? @nilminiwso2 @isuruw any comment on this?
> 
> Triggers and Scheduled Jobs are also components, since everything in Choreo is a component. These components are integration and automation specific that is why we categorised those separately. I also agree to use the word "Component" rather than "Feature" here, as we dont use the term "feature" anywhere.
> 
> Triggers and Scheduled Jobs are also bound to the component limit, if you have 3 proxy APIs and 2 scheduled jobs then your limit is reached. And they are free until the component limit is not exceeded.
> 
> Therefore for line 88 , I suggest to use as "integration and automation components"

But then, connectors are listed in the same section. If we change to "integration and automation components" they too will be listed 

> > About comment "It can be confusing, as we are referring to them as components, and when we come here it is features. Can give a wrong idea that Triggers and scheduled jobs are free" on line 88.
> > Well, they are free as long as you don't exceed the component quota isn't it? The idea is that the trigger and scheduled job functionality is available for all payment plans? @nilminiwso2 @isuruw any comment on this?
> 
> Triggers and Scheduled Jobs are also components, since everything in Choreo is a component. These components are integration and automation specific that is why we categorised those separately. I also agree to use the word "Component" rather than "Feature" here, as we dont use the term "feature" anywhere.
> 
> Triggers and Scheduled Jobs are also bound to the component limit, if you have 3 proxy APIs and 2 scheduled jobs then your limit is reached. And they are free until the component limit is not exceeded.
> 
> Therefore for line 88 , I suggest to use as "integration and automation components"

Changed this section as follows to avoid categorizing the connectors also as components. Hope this is ok.

```
#### Integration and automation

Choreo supports the following integration and automation components for all the plans:

- Triggers
- Scheduled jobs

In addition, Choreo supports 500+ out-of-the-box connectors for integration and automation for all the plans.
```
